### ISSUE TITLE
"There was a problem checking your Vagrant version"

### ISSUE THREAD

**Issue by CodeGnome** [lanayotech/vagrant-manager#158]
Using Vagrant Manager 2.5.4 with Vagrant 1.9.7, checking for vagrant updates through the Vagrant Manager GUI always results in the following error:

> There was a problem checking your Vagrant version

![screen shot 2017-07-13 at 2 02 14 pm](https://user-images.githubusercontent.com/385462/28180803-43336324-67d4-11e7-9427-b237415f83bb.png)

Both Vagrant and Vagrant Manager were installed via HomeBrew. The system is running macOS 10.12.5, and is replicable 100% of the time even after removing and reinstalling Vagrant Manager.

**Comment by chriswayoub** [comment]
Are you still having this issue? I haven't been able to recreate it. Can you try the [latest release](https://github.com/lanayotech/vagrant-manager/releases/tag/2.6.0)?

**Comment by CodeGnome** [comment]
Sort of. As of Version 2.6.0 on High Sierra checking for updates now just goes off into la-la land. If you check for updates, the application becomes unresponsive. For example, Vagrant Manager will no longer display the About screen after clicking Check for Updates.

**Comment by chriswayoub** [comment]
I believe you are now running into issue #149. Can you check if the check for updates window is actually just hidden behind your other windows? It won't show up in Exposé or your window list so you'll have to move your other windows out of the way to find it.

**Comment by CodeGnome** [comment]
Checking for Vagrant Manager updates produces a window, but checking for Vagrant updates through the GUI still results in the same error as before. Z-order doesn't appear to be the core issue here.

![screen shot 2017-11-16 at 2 02 38 pm](https://user-images.githubusercontent.com/385462/32911821-5292c142-cadb-11e7-8aae-08c891af9e2d.png)

**Comment by chriswayoub** [comment]
What Vagrant version are you on?

**Comment by CodeGnome** [comment]
```
$ vagrant --version
Vagrant 2.0.1
```

**Comment by chriswayoub** [comment]
Can you paste your output from `vagrant version --machine-readable`? Are you using a shell other than bash?

**Comment by CodeGnome** [comment]
The command above failed at first. Next, I ran `vagrant plugin repair`, and then the Bash prompt returned:
```
$ vagrant version --machine-readable
1511286485,,ui,output,Installed Version: 2.0.1
1511286485,,version-installed,2.0.1
1511286485,,ui,output,Latest Version: 2.0.1
1511286485,,version-latest,2.0.1
1511286485,,ui,success, \nYou're running an up-to-date version of Vagrant!
```
However, Vagrant Manager refused to respond to "Check for Vagrant Updates" until it was killed from the command line ("Quit" wouldn't work) and then restarted.

One the plugins were repaired and Vagrant Manager restarted, Vagrant Manager would again properly report that everything was at the current version. However, the dialog window is typically at the bottom of the z-order as described by the other bug.

![screen shot 2017-11-21 at 12 50 48 pm](https://user-images.githubusercontent.com/385462/33088262-313a472a-cebb-11e7-8b9f-1af5b15b9b00.png)

Expected behavior would be for Vagrant Manager to report *something* rather than just not reporting anything at all when there's a problem with plugins or a change in version while Vagrant Manager is running. So, even though I've fixed my own problem for now, I'd respectfully suggest that this is still a bug.

**Comment by chriswayoub** [comment]
I agree, and I'm not sure why it would just hang. It should at least give you back an error if it couldn't parse the output of that command, so I'll look into why it might not. When the window opens behind your other windows, it does cause Vagrant Manager to hang and stop responding to any commands until that window is closed. Is it possible the window letting you know "there was a problem checking the version" was open somewhere behind your other windows?

**Comment by CodeGnome** [comment]
Nope. There is no window at all in that case.
